spec/dfe/analytics/services/entity_table_checks_spec.rb

```diff
@@ -68,7 +67,11 @@
     let(:department_entity) { DfE::Analytics.entities_for_analytics.find { |entity| entity.to_s.include?('department') } }
     let(:entity_type) { 'entity_table_check' }
```

Comment:
Use instance variables in tests is normally discouraged. I would add a let for the checksum_calculated_at instead of using an instance variable in the tests.
```
  let(:checksum_calculated_at) { @checksum_calculated_at }
```

Then just use  checksum_calculated_at rather than @checksum_calculated_at